source/_lovelace/entity-button.markdown

```diff
@@ -110,6 +115,11 @@ hold_action:
       description: "Service data to include (e.g. `entity_id: media_player.bedroom`) when `action` defined as `call-service`"
       type: string
       default: none
+    confirmation:
+      required: false
+      description: "Present a confirmation dialog to confirm the action. See `confirmation` object below"
+      type: [boolean, map]
+      default: "false"
```

Comment:
You don't have to quote booleans ;)

Reply:
All other defaults in this file had quotes, so I followed the trend 🤷‍♂